mmhuman3d/data/data_converters/agora.py

```diff
@@ -21,17 +21,27 @@ class AgoraConverter(BaseModeConverter):
     Args:
         modes (list): 'validation' or 'train' for accepted modes
         fit (str): 'smpl' or 'smplx for available body model fits
+        res (str): '1280x720' or '3840x2160' for available image resolution
     """
     ACCEPTED_MODES = ['validation', 'train']
 
-    def __init__(self, modes: List = [], fit: str = 'smpl') -> None:
+    def __init__(self,
+                 modes: List = [],
+                 fit: str = 'smpl',
+                 res='1280x720') -> None:
         super(AgoraConverter, self).__init__(modes)
         accepted_fits = ['smpl', 'smplx']
         if fit not in accepted_fits:
             raise ValueError('Input fit not in accepted fits. \
                 Use either smpl or smplx')
         self.fit = fit
 
+        accepted_res = ['1280x720', '3840x2160']
```

Comment:
May use a tuple (e.g. (1280, 720) ) for resolution instead of a string

Reply:
done